registration/include/pcl/registration/impl/ndt.hpp

```diff
@@ -301,12 +293,12 @@ NormalDistributionsTransform<PointSource, PointTarget>::computePointDerivatives
     Eigen::Matrix<double, 15, 1> point_angular_hessian = angular_hessian_ * Eigen::Vector4d(x[0], x[1], x[2], 0.0);
 
     // Vectors from Equation 6.21 [Magnusson 2009]
-    Eigen::Vector3d a(0, point_angular_hessian[0], point_angular_hessian[1]);
-    Eigen::Vector3d b(0, point_angular_hessian[2], point_angular_hessian[3]);
-    Eigen::Vector3d c(0, point_angular_hessian[4], point_angular_hessian[5]);
-    Eigen::Vector3d d(point_angular_hessian[6], point_angular_hessian[7], point_angular_hessian[8]);
-    Eigen::Vector3d e(point_angular_hessian[9], point_angular_hessian[10], point_angular_hessian[11]);
-    Eigen::Vector3d f(point_angular_hessian[12], point_angular_hessian[13], point_angular_hessian[14]);
+    const Eigen::Vector3d a(0, point_angular_hessian[0], point_angular_hessian[1]);
+    const Eigen::Vector3d b(0, point_angular_hessian[2], point_angular_hessian[3]);
+    const Eigen::Vector3d c(0, point_angular_hessian[4], point_angular_hessian[5]);
+    const Eigen::Vector3d d(point_angular_hessian[6], point_angular_hessian[7], point_angular_hessian[8]);
+    const Eigen::Vector3d e(point_angular_hessian[9], point_angular_hessian[10], point_angular_hessian[11]);
+    const Eigen::Vector3d f(point_angular_hessian[12], point_angular_hessian[13], point_angular_hessian[14]);
```

Comment:
These are sequential index accesses. Can we access them in a block?

Reply:
Done